src/cc/bpf_module.cc

```diff
@@ -101,6 +101,7 @@ class MyMemoryManager : public SectionMemoryManager {
 
 BPFModule::BPFModule(unsigned flags, TableStorage *ts)
     : flags_(flags),
+      rw_engine_enabled_(true),
```

Comment:
Can we do this via `flags`? like `rw_engine_enabled_(!(flags & BPF_DISABLE_RW_ENGINE))`? This way it would be easy to control the behavior through Python / C++ API (who already has `flags` as constructor argument of the `BPF` class) without need to change the interface?